api-spec/AuthEndpoints.md

```diff
@@ -167,3 +173,18 @@ PATH: `GET user/verify/:secret_key`
 
 #### `401 Unauthorized`
 > The secret key is invalid or expired.
+
+## `GET user/create_secret/:user_id`
```

Comment:
Please add the validate route to this as well

Reply:
That's connor's thing. I don't know how it got included in this pr. I can't do anything about it either.